### ISSUE TITLE
package.xml in wrong directory (noetic on focal)

### ISSUE THREAD

**Issue by otherwiseopen** [roboception-gbp/rcdiscover-release#1]
When installing this package in focal-noetic (in a docker image based on ros:noetic-robot), ROS does not work correctly, because the package places a `package.xml` into `/opt/ros/noetic/share/`

Checking the package contents shows:
```
root@ad01427929ed:~# dpkg -L ros-noetic-rcdiscover
/.
/opt
/opt/ros
/opt/ros/noetic
/opt/ros/noetic/bin
/opt/ros/noetic/bin/rcdiscover
/opt/ros/noetic/include
/opt/ros/noetic/include/rcdiscover
/opt/ros/noetic/include/rcdiscover/deviceinfo.h
/opt/ros/noetic/include/rcdiscover/discover.h
/opt/ros/noetic/include/rcdiscover/force_ip.h
/opt/ros/noetic/include/rcdiscover/gige_request_counter.h
/opt/ros/noetic/include/rcdiscover/operation_not_permitted.h
/opt/ros/noetic/include/rcdiscover/ping.h
/opt/ros/noetic/include/rcdiscover/socket.h
/opt/ros/noetic/include/rcdiscover/socket_exception.h
/opt/ros/noetic/include/rcdiscover/socket_linux.h
/opt/ros/noetic/include/rcdiscover/utils.h
/opt/ros/noetic/include/rcdiscover/wol.h
/opt/ros/noetic/include/rcdiscover/wol_exception.h
/opt/ros/noetic/lib
/opt/ros/noetic/lib/x86_64-linux-gnu
/opt/ros/noetic/lib/x86_64-linux-gnu/librcdiscover.so.1.1
/opt/ros/noetic/lib/x86_64-linux-gnu/rcdiscover
/opt/ros/noetic/lib/x86_64-linux-gnu/rcdiscover/RCDISCOVERConfig.cmake
/opt/ros/noetic/lib/x86_64-linux-gnu/rcdiscover/RCDISCOVERConfigVersion.cmake
/opt/ros/noetic/lib/x86_64-linux-gnu/rcdiscover/RCDISCOVERTargets-none.cmake
/opt/ros/noetic/lib/x86_64-linux-gnu/rcdiscover/RCDISCOVERTargets.cmake
/opt/ros/noetic/share
/opt/ros/noetic/share/package.xml
/opt/ros/noetic/share/rcdiscover
/opt/ros/noetic/share/rcdiscover/50-rcdiscover-rpfilter.conf
/usr
/usr/share
/usr/share/doc
/usr/share/doc/ros-noetic-rcdiscover
/usr/share/doc/ros-noetic-rcdiscover/changelog.Debian.gz
/opt/ros/noetic/lib/x86_64-linux-gnu/librcdiscover.so
```
this prevents ROS from working correctly.
The file should probably go into `/opt/ros/noetic/share/rcdiscover/`.

**Comment by flixr** [comment]
~~This is caused by https://github.com/ros-infrastructure/bloom/issues/642~~

I will update it so that at least the package.xml is always installed to share...
In the meantime you could [build rcdiscover](https://github.com/roboception/rcdiscover#building-rcdiscover) yourself and install as deb with
```
cmake  -DCMAKE_INSTALL_PREFIX="/opt/ros/$ROS_DISTRO" -DCMAKE_PREFIX_PATH="/opt/ros/$ROS_DISTRO" -DCMAKE_BUILD_TYPE=Release ..
make package
sudo dpkg -i rcdiscover*.deb
```
which doesn't trigger this bug.

**Comment by flixr** [comment]
Sorry, has nothing to do with bloom GnuInstallDirs bug...
Fix coming up..

**Comment by flixr** [comment]
Fixed with [rcdiscover 1.1.3](https://github.com/roboception/rcdiscover/commit/80d24972a0be49f514849f896a04301c81c38c6e)
Blooming now, will take a while until this is rebuilt on the buildfarm and included in the next sync.
In the meantime you can build the deb yourself as described above.
